zebra-network/src/peer_set/inventory_registry.rs

```diff
@@ -0,0 +1,37 @@
+use crate::protocol::external::InventoryHash;
+use std::{
+    collections::{HashMap, HashSet},
+    net::SocketAddr,
+};
+
+#[derive(Debug, Default)]
+pub(super) struct InventoryRegistry {
```

Comment:
Together with the change in https://github.com/ZcashFoundation/zebra/pull/963/files#r479588057 , I think it would be better for this to be `pub` rather than `pub(super)`, so that the item visibility is controlled in one place without exposing details of the `inventory_registry` module to the sibling modules.  In general I think that `pub(super)` is better to avoid if possible, because although it's useful in cases where sibling modules aren't encapsulated from each other, it's not necessary when they are encapsulated from each other, and that is usually better code organization.